target/linux/ramips/dts/mt7621_netgear_sercomm_bzv.dtsi

```diff
@@ -0,0 +1,230 @@
+// SPDX-License-Identifier: GPL-2.0
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "mediatek,mt7621-soc";
+
+	aliases {
+		led-boot = &led_power_white;
+		led-failsafe = &led_power_orange;
+		led-running = &led_power_white;
+		led-upgrade = &led_power_orange;
+		label-mac-device = &ethernet;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,57600";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_orange: power_orange {
+			gpios = <&gpio_expander 0 GPIO_ACTIVE_LOW>;
+		};
+
+		led_power_white: power_white {
+			gpios = <&gpio_expander 1 GPIO_ACTIVE_LOW>;
+		};
+
+		led_wan_orange: wan_orange {
+			gpios = <&gpio_expander 2 GPIO_ACTIVE_LOW>;
+		};
+
+		led_wan_white: wan_white {
+			gpios = <&gpio_expander 3 GPIO_ACTIVE_LOW>;
+		};
+
+		led_wifi2g: wifi2g {
+			gpios = <&gpio_expander 4 GPIO_ACTIVE_LOW>;
+		};
+
+		led_wifi5g: wifi5g {
+			gpios = <&gpio_expander 5 GPIO_ACTIVE_LOW>;
+		};
+
+		led_usb2: usb2 {
+			gpios = <&gpio_expander 6 GPIO_ACTIVE_LOW>;
+		};
+
+		led_usb3: usb3 {
+			gpios = <&gpio_expander 7 GPIO_ACTIVE_LOW>;
+		};
+
+		led_lan1_orange: lan1_orange {
+			gpios = <&gpio_expander 8 GPIO_ACTIVE_LOW>;
+		};
+
+		led_lan1_white: lan1_white {
+			gpios = <&gpio_expander 9 GPIO_ACTIVE_LOW>;
+		};
+
+		led_lan2_orange: lan2_orange {
+			gpios = <&gpio_expander 10 GPIO_ACTIVE_LOW>;
+		};
+
+		led_lan2_white: lan2_white {
+			gpios = <&gpio_expander 11 GPIO_ACTIVE_LOW>;
+		};
+
+		led_lan3_orange: lan3_orange {
+			gpios = <&gpio_expander 12 GPIO_ACTIVE_LOW>;
+		};
+
+		led_lan3_white: lan3_white {
+			gpios = <&gpio_expander 13 GPIO_ACTIVE_LOW>;
+		};
+
+		led_lan4_orange: lan4_orange {
+			gpios = <&gpio_expander 14 GPIO_ACTIVE_LOW>;
+		};
+
+		led_lan4_white: lan4_white {
+			gpios = <&gpio_expander 15 GPIO_ACTIVE_LOW>;
+		};
+
+		led_wps: wps {
+			gpios = <&gpio0 17 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_wifi: wifi {
+			gpios = <&gpio0 5 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		wps {
+			label = "wps";
+			gpios = <&gpio0 18 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+
+		rfkill {
+			label = "rfkill";
+			gpios = <&gpio0 14 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RFKILL>;
+		};
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 12 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+
+&pcie {
+	status = "okay";
+};
+
+&pcie0 {
+	wifi@0,0 {
+		compatible = "mediatek,mt76";
+		reg = <0x0 0 0 0 0>;
+		mediatek,mtd-eeprom = <&factory 0x0>;
+		ieee80211-freq-limit = <2400000 2500000>;
+	};
+};
+
+&pcie1 {
+	wifi@0,0 {
+		compatible = "mediatek,mt76";
+		reg = <0x0 0 0 0 0>;
+		mediatek,mtd-eeprom = <&factory 0x8000>;
+		ieee80211-freq-limit = <5000000 6000000>;
+	};
+};
+
+&ethernet {
+	mtd-mac-address = <&factory 0x4>;
+};
+
+&pinctrl {
+	state_default: pinctrl0 {
+		gpio {
+			ralink,group = "uart3", "uart2", "jtag", "wdt", "spi";
+			ralink,function = "gpio";
+		};
+	};
+};
```

Comment:
state_default is already defined in mt7621.dtsi, so you use it here:

```
&state_default {
	gpio {
		ralink,group = "uart3", "uart2", "jtag", "wdt", "spi";
		ralink,function = "gpio";
	};
};
```

Haven't found why you need "spi" here?

Reply:
Sorry. I forgot remove it.

Reply:
Still need to change that to:
```
&state_default {
	gpio {
		ralink,group = "uart3", "uart2", "jtag", "wdt";
		ralink,function = "gpio";
	};
};
```